posthog/test/test_instance_setting_model.py

```diff
@@ -67,3 +69,22 @@ def test_can_retrieve_multiple_settings(db):
         "MATERIALIZED_COLUMNS_ENABLED": True,
         "ASYNC_MIGRATIONS_AUTO_CONTINUE": True,
     }
+
+
+@pytest.mark.parametrize("cached_value", ("true", '["1:cool","abc:123"]', '"a_string"', "123"))
+def test_get_cached_instance_setting(db, cache, cached_value):
```

Comment:
Can we have a test specifically for the capture endpoint and setting the setting via the Postgres db.